src/gui/owncloudgui.h

```diff
@@ -64,10 +65,13 @@ class ownCloudGui : public QObject
     bool cloudProviderApiAvailable();
 #endif
     void createTray();
+    void init();
```

Comment:
I suspect it might not be necessary once we step back from a singleton, to be confirmed.